.github/workflows/test_integration.yml

```diff
@@ -49,7 +49,7 @@ jobs:
           github-token: ${{ steps.get_token.outputs.token }}
           script: |
             const script = require('.github/workflows/scripts/test_integration.js')
-            await script.post_status_check(github, context, "pending")
+            await script.post_status_check(github, context, "integration", "pending")
```

Comment:
This step can be removed since we submit the pending status in the spec repo now

Reply:
it submits a better status URL

Reply:
Ah right ! 👍